rewrite-java/src/main/java/org/openrewrite/java/ChangeType.java

```diff
@@ -199,6 +224,19 @@ public J visitImport(J.Import import_, ExecutionContext ctx) {
             return updateType(javaType);
         }
 
+        @Override
+        public J visitLiteral(J.Literal literal, ExecutionContext ctx) {
+            J.Literal lit = literal;
+            if (Boolean.TRUE.equals(visitLiterals) && literal.getType() == JavaType.Primitive.String) {
+                Pattern pat = Pattern.compile("(?:\\A|\\s)" + oldFullyQualifiedTypeName + "(?:|\\s)");
```

Comment:
Are we not needlessly complicating things here? I'm usually a fan of using [word boundaries with `\b`](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) , as that matches a whole category:
```suggestion
                Pattern pat = Pattern.compile("\\b" + oldFullyQualifiedTypeName + "\\b");
```